papers/bib.py

```diff
@@ -1140,14 +1140,14 @@ def listcmd(o):
         entries = my.db.entries
 
         if o.fuzzy:
-            from fuzzywuzzy import fuzz
+            from rapidfuzz import fuzz
 
         def match(word, target, fuzzy=False, substring=False):
             if isinstance(target, list):
                 return any([match(word, t, fuzzy, substring) for t in target])
 
             if fuzzy:
-                res = fuzz.token_set_ratio(word.lower(), target.lower()) > o.fuzzy_ratio
+                res = fuzz.token_set_ratio(word, target, score_cutoff=o.fuzzy_ratio) > o.fuzzy_ratio
```

Comment:
strings are lowercased automatically both by rapidfuzz and fuzzywuzzy already. the score cutoff parameter in rapidfuzz allows it to exit early when it is sure it can not reach the cutoff score